build.properties

```diff
@@ -2,10 +2,8 @@
 # Don't directly modify this file. Instead, copy it to local.build.properties and
 # edit that.
 #
-# $Id$
 project.name = eXist-db
-project.version = 3.0
```

Comment:
I remove `project.version.numeric` because I did not found any references to it. If external application depend on that property then I can change it.